homeassistant/components/google_assistant/services.yaml

```diff
@@ -1,2 +1,5 @@
 request_sync:
-  description: Send a request_sync command to Google.
\ No newline at end of file
+  description: Send a request_sync command to Google.
+  fields:
+    agent_user_id:
+      description: [Optional] specific Home Assistant user id to sync with Google Assistant. Do no need when you call this services through Home Assistant front end or API. Used in automation script or other place where context.user_id is missing.
```

Comment:
Brackets will fail yaml and produce a traceback.  In testing I removed the brackets and this will load.